source/robot_model/include/robot_model/Model.hpp

```diff
@@ -208,7 +214,14 @@ inline void Model::set_urdf_path(const std::string& urdf_path) {
 
 inline unsigned int Model::get_nb_joints() const {
```

Comment:
btw is `nb` short for "number"? That's not a common abbreviation in my opinion. Maybe it could be refactored to `get_number_of_joints` or the very verbose `get_joint_degrees_of_freedom` or the very sparse `get_nq`. Just some thoughts, I don't have a strong preference but I don't think `nb_joints` is very clear on its own.

Reply:
yes make sense. I think the verbose `get_number_of_joints` is the better choice here